project/.gitignore

```diff
@@ -29,3 +29,4 @@ bin/
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 /target/
+src/main/java/de/unistuttgart/informatik/fius/jvk/Main1.java
```

Comment:
Can you explain this?
Is this just a temporary Main file you used to test the code?

Reply:
Yes, it was for temporary testing purposes. I forgot to change it afterwards.